.yarnrc.yml

```diff
@@ -1 +1,6 @@
 yarnPath: .yarn/releases/yarn-3.2.1.cjs
+packageExtensions:
+  chalk@5.0.1:
+    dependencies:
+      '#ansi-styles': npm:ansi-styles@6.1.0
+      '#supports-color': npm:supports-color@9.2.2
```

Comment:
What do we plan to use this for? Logging?

Reply:
If we don't have this we can't run `serve` -  which I was using to test in scigateway